handlers/api.go

```diff
@@ -3470,25 +3501,37 @@ func getAuthClaims(r *http.Request) *utils.CustomClaims {
 	return claims.(*utils.CustomClaims)
 }
 
-func returnQueryResults(rows *sql.Rows, w http.ResponseWriter, r *http.Request) {
+func returnQueryResults(rows *sql.Rows, w http.ResponseWriter, r *http.Request, adjustQueryEntriesFuncs ...func(map[string]interface{}) error) {
```

Comment:
**Minor:** The new parameter is not really self explanatory, maybe add a comment above the function here and here:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/817b4c30dfccb52a0821a6106467af78cd5e0766/handlers/api.go#L3521

Reply:
Done.